Dockerfile

```
@@ -1,9 +1,9 @@
-FROM ubuntu
+FROM ubuntu:20.04
```

I suggest to change it with "latest" since latest means the latest LTS of Ubuntu. 

> The ubuntu:latest tag points to the "latest LTS", since that's the version recommended for general use. The ubuntu:rolling tag points to the latest release (regardless of LTS status).

https://hub.docker.com/_/ubuntu

```suggestion
FROM ubuntu:latest
```

---

This program return 'No usable version of libssl was found' at Ubuntu 22.04 LTS, so use Ubuntu 20.04 LTS instead.